### PR TITLE
Adjust the deploymet job for ECR images

### PR DESCRIPTION
**Pull request by higs4281**
As part of this refactor, required by a change in Docker registries,
we drop the json-as-env-var structure, which was brittle, and
build the json configuration in the python script.

Sensitive bits come from Jenkins env vars.

cc @dvbalen

### CODE REVIEW

**Review by KatherynRizek — APPROVED** [review]
I'm not the most familiar with this process but nothing seems glaringly wrong. If there are particular testing steps I should take, please let me know!

**Comment by higs4281** [comment]
@KatherynRizek Thanks for the eyes. I hope to spread awareness of the code that deploys our app.

**Comment by higs4281** [comment]
For posterity: This change won't have immediate effect on most active jobs because most point at various branches of a personal fork of this repo created by a departed staffer: https://github.com/tanderegg/marathon-cli

As we update jobs, we can point them at this repo to use a centralized json form for apps.

**Review by imuchnik — CHANGES_REQUESTED** [review]

File: marathon-cli.py [hunk 1]
```diff
@@ -39,56 +52,184 @@ def print_file_chunk(url, offset, auth):
...
+        "cpus": 2,
+        "mem": 8000,
+        "disk": 5000,
+        "instances": 2,
+        "uris": [],
+        "user": f'{os.getenv("DOCKER_USER")}',
+        "env": {
+            "ES_SERVER_ENV": "staging",
+            "DJANGO_SETTINGS_MODULE": "search_tool.mesos",
+            "ES_HOST": "https://es2.data.cfpb.local/",
```
[the remark this answers is not in the document]
Comment (higs4281) [reply marathon-cli.py:151]: good catch. Now passed from Jenkins.

**Review by imuchnik — APPROVED** [review]

**Pull request merged**